test/common.js

```diff
@@ -12,6 +12,61 @@ exports.tmpDirName = 'tmp';
 exports.PORT = +process.env.NODE_COMMON_PORT || 12346;
 exports.isWindows = process.platform === 'win32';
 
+var rimrafSync = function(p) {
```

Comment:
OK, new commit pushed with the function declarations changed to conform with the prevailing idiom already in the file!